docs/documentation/features/internationalization.md

```diff
@@ -83,29 +83,46 @@ XML file (thing-types.xml):
     <thing-type id="weather">
         <label>Weather Information</label>
         <description>Provides various weather data from the Yahoo service</description>
-		<channels>
-			<channel id="temperature" typeId="temperature" />
-		</channels>
+        <channels>
+            <channel id="unit-system" typeId="unit-system" />
```

Comment:
That's a pretty bad example as such a feature should rather be a config parameter and not a channel and because we are about to change the whole unit support on channels soon - so for both reasons, this can easily be misleading and outdated soon...

How about defining an imaginary channel "precipitation" with options "dry", "raining" and "snowing"?